<launch>
 <!-- Turtlesim Node-->
 <!-- <node pkg="turtlesim" type="turtlesim_node" name="sim"/> -->

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- rosserial node -->
  <node pkg="rosserial_python" type="serial_node.py" name="rosserial_node">
    <param name="~port" value="/dev/ttyACM0" />
    <param name="~baud" value="57600" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>
  <node pkg="corobot_motor_control" type="joy_control2" name="teleop"/>
</launch>
